bob/src/hw_metrics_collector.rs

```diff
@@ -146,10 +149,14 @@ impl DescrCounter {
         //  with payload
         //  |  10.000   |      0.018     |
         let d = std::fs::read_dir(DESCRS_DIR);
-        if let Ok(d) = d {
-            d.count() as u64 - 4 // exclude stdin, stdout, stderr and `read_dir` instance
-        } else {
-            0 // proc is unsupported
+        match d {
+            Ok(d) => {
+                d.count() as u64 - 4 // exclude stdin, stdout, stderr and `read_dir` instance
+            }
+            Err(e) => {
+                warn!("failed to count descriptors: {}", e);
```

Comment:
If `/proc/` is not available, this message will clog up the log file. Since descriptors counting is not a critical feature, it is better to print this log on `debug` level